### DISCUSSION TITLE
Remove languages with only one directory or not very much pages

### DISCUSSION THREAD

**Discussion by sebastiaanspeck** [tldr-pages/tldr#11512] [Polls]
Hi,

At tldr we support a lot of languages, thanks to our community, but this also comes with responsibility to maintain and keeping the language up-to-date. There are several languages that don't have a [CODEOWNER](https://github.com/tldr-pages/tldr/blob/main/.github/CODEOWNERS) or just don't get any new pages etc. IMO this creates clutter and might even affect the quality of tldr.

The languages I suggest to remove are:
- https://github.com/tldr-pages/tldr/tree/main/pages.cs
- https://github.com/tldr-pages/tldr/tree/main/pages.fi
- https://github.com/tldr-pages/tldr/tree/main/pages.ro
- https://github.com/tldr-pages/tldr/tree/main/pages.sh
- https://github.com/tldr-pages/tldr/tree/main/pages.sr
- https://github.com/tldr-pages/tldr/tree/main/pages.uz

**Comment by sbrl** [comment]
I *strongly* suggest avoiding polls, @sebastiaanspeck. It is important that instead of going with what the majority think, to talk it out and come to a consensus. This might take longer, but ensures that we make better decisions and everyone's voice is heard.

To answer your question, surely a translated page that might be out of date is better than nothing? It's more inclusive I think to keep them. This is probably a part of a wider discussion on maintaining translations. See also #3591, which I believe is the most recent discussion on the subject.

See point 4 in [GOVERNANCE.md](https://github.com/tldr-pages/tldr/blob/main/GOVERNANCE.md).
